docs/install.md

```diff
@@ -209,10 +209,25 @@ on the Kubernetes version.
 
 - **Label the cluster nodes that provide persistent memory device(s)**
 
+The local part of PMEM-CSI runs on all nodes with a certain label. For
```

Comment:
@pohly What exactly 'local' refers to here?

Reply:
The node server.

Saying the "The node part of PMEM-CSI runs on all nodes with a certain label" felt odd.

Reply:
Replaced with "PMEM-CSI manages PMEM on those nodes that have a certain label."